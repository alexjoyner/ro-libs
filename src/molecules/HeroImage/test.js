import React from 'react';
import { shallow } from 'enzyme';
import { HeroImage } from '.';

describe('HeroImage component', () => {
  let component;
  beforeEach(() => {
    component = shallow(<HeroImage />);
  });
  it('Should render without exploding', () => {
    expect(component).toBeDefined();
  });
  it('Should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
