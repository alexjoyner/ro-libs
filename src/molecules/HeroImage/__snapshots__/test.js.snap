// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroImage component Should match snapshot 1`] = `
<Wrapper>
  <ImageOverlay>
    <HeroImageContent />
  </ImageOverlay>
</Wrapper>
`;
